src/gkmodel/model.py

```diff
@@ -58,6 +58,16 @@ def predict(self, data: MRData = None):
         pred1 = self.model1.predict(data)
         return self.model2.predict(data) + pred1
 
+    def write_stage1_soln(self):
```

Comment:
Since the function is `write`, could we also have an keyword argument `path`, if it is not `None` also save the data frame as csv file?